earth_enterprise/legacy_installer/README.txt

```diff
@@ -0,0 +1 @@
+This is the obsolete installer. See ../src/installer for the scripts you want.
```

Comment:
I don't want to have `earth_enterprise/installer` and `earth_enterprise/src/installer` because its so easy to confuse them (results in developer friction).

Can you explain "Aaron-installer" - that doesn't mean anything to me.